src-electron/upgrade/upgrade.js

```diff
@@ -40,14 +40,42 @@ async function getForcedExternalStorage(db, attributeId) {
   let pkgs = await queryPackage.getPackageRefByAttributeId(db, attributeId)
   let zcl = await queryPackage.getPackageByPackageId(db, pkgs)
   zcl = zcl.path
-  let obj = await fsp.readFile(zcl)
-  let data = JSON.parse(obj)
+  let obj = await fsp.readFile(zcl, 'utf-8')
+  let data
+  data = JSON.parse(obj)
```

Comment:
I don't understand this change now....  Why are we declaring and assigning on different lines?